src/Relation/Binary/Bundles.agda

```diff
@@ -297,3 +297,14 @@ record StrictTotalOrder c ℓ₁ ℓ₂ : Set (suc (c ⊔ ℓ₁ ⊔ ℓ₂)) wh
   "Warning: decSetoid was deprecated in v1.3.
   Please use Eq.decSetoid instead."
   #-}
+
+
+
+record Apartness c ℓ₁ ℓ₂ : Set (suc (c ⊔ ℓ₁ ⊔ ℓ₂)) where
```

Comment:
This should go above the deprecated names in the file!

Reply:
Hang on I think that deprecated name is local to `StrictTotalOrder`. ;-)

Reply:
Yes, you're right! My fault for not expanding upwards and only relying on the limited Github view.